SECURITY.md

```diff
@@ -8,8 +8,8 @@ The following table shows which versions of `price-oracle` are currently being s
 
 | Version     | Supported          |
 |-------------|--------------------|
-| `v0.10.3`   | :white_check_mark: |
-| `< v0.10.3` | :x:                |
+| `v0.10.4`   | :white_check_mark: |
+| `< v0.10.4` | :x:                |
```

Comment:
Isn't this supposed to be `v0.10.5`? Since this doc represents the version of price oracle repo

Reply:
yes, thanks!